app/assets/stylesheets/application.scss

```diff
@@ -2,13 +2,92 @@
 @import "bootstrap-sprockets";
 @import "bootstrap";
 
+body {
+  background: #F9F9F9
```

Comment:
Color literals like `#F9F9F9` should only be used in variable declarations; they should be referred to via variable everywhere else.<br>Color `#F9F9F9` should be written as `#f9f9f9`<br>Declaration should be terminated by a semicolon